### DISCUSSION TITLE
Preview for qmd files in VS code.

### DISCUSSION THREAD

**Discussion by Adam-Antios** [quarto-dev/quarto-cli#4381] [Feature Requests]
In the case of .md files, VS Code has a live preview feature at hand without the need for launching a server. It is easy to work on the files without a fuss.

On the other hand, Quarto has the preview server, but as a workflow it is more cumbersome when you simply want to view a markdown file you work with. Moreover, the quarto preview actually creates an html file along with miscellaneous resources.

Given that .qmd files are markdown, can't you activate the same functionality? Obviously the most advanced features will not be rendered in the live preview but there will be at least the base functionality.

**Comment by cderv** [comment]
See here about this : https://github.com/quarto-dev/quarto/issues/89

Thanks for the feedback

**Comment by mcanouil** [comment]
Closing as completed in https://github.com/quarto-dev/quarto/issues/89
